lib/fog/libvirt/models/compute/server.rb

```diff
@@ -1,6 +1,7 @@
 require 'fog/compute/models/server'
 require 'fog/libvirt/models/compute/util/util'
 require 'fileutils'
+require 'tempfile'
```

Comment:
Without this fog-libvirt throws `uninitialized constant Fog::Libvirt::Compute::Server::Tempfile`. Might have something to do with Ruby version... (I am on `ruby 2.6.3p62`)